nilearn/image/resampling.py

```diff
@@ -306,6 +306,11 @@ def resample_img(img, target_affine=None, target_shape=None,
         input image, resampled to have respectively target_shape and
         target_affine as shape and affine.
 
+    See Also
+    --------
+    nilearn.resampling.resample_to_img : To resample an image on a target
+    image.
```

Comment:
Did you check that this renders well. I have doubt on the indentation in seealso for multiple lines.